text-editor/text-editor-002-en.md

```diff
@@ -581,5 +581,76 @@ compatible with the contents edited by the legacy app.
 This brings us to the `sanitization` step, where we replace any rendered
 form of complex inserted media by their embed form. To do so, we have to
 keep the embed representation of the media in the application state. We also
-have to identify to which rendered form correspond an embed markup (eg for
-tweets, we're using the tweet id).
+have to identify to which rendered form corresponds an embed markup (eg for
+tweets, we're using the tweet id to make this matching).
```

Comment:
Maybe avoid the generic `form` term, as it can be confused with a form element ?

Reply:
fixed ;)